src/main/kotlin/org/jitsi/rtp/NewRawPacket.java

```diff
@@ -1058,300 +1058,31 @@ public HeaderExtension next()
 
 class RTPUtils
```

Comment:
RTPUtils now only has generic methods for writing to a byte array. It is meant to dissapear in the future, right?

Reply:
yup